spec/requests/api/v1/keywords_controller_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V1::KeywordsController, type: :controller do
+  describe 'GET#index' do
+    it 'returns expected status' do
+      get :index
+
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'returns response with valid keyword' do
```

Comment:
We can use different verb to make the sentence more concise.

```suggestion
    it 'responds the valid keyword' do
```